c09 translation (#194)
IllusionMods/KKS-Translation: Translation/en/RedirectedResources/assets/abdata/communication/info_00/topictalkrare_09/translation.txt

「ムラムラする時？　触って欲しい時とかは、自分でしてるのかなって」=“Like when you're horny? I was just wondering if you touch yourself when you start craving it or whatever.”
「す、するわけないでしょ！　そんなことしたらエッチな子じゃない」=“O-Of course I don't! I'd be such a lewd girl if I did that stuff.”
「じゃあ、我慢するんだ？」=“You endure it then?”
「そうよ、ムラムラしても我慢。あたしは並外れた忍耐力があるから」=“That's right, I endure it even when I feel horny. I've got the devil's endurance, you know.”
（ムラムラすることはあるんだ……）=(So she does get horny, huh...)
「どうしても意識しちゃうな。俺達、その……しちゃったんだよね？」=“It's hard to not think about it, you know? We really, uhm... did that, didn't we?”
「はっきり言わないでよ。顔、熱くなっちゃうから考えないようにしてたのに」=“Don't say it so directly. I've been trying not to think about it too, since it makes my face all red.”
「俺もそうしたいんだけど、どうしても思い出しちゃって」=“I want to do the same but, I just can't help thinking back to it.”
「どのシーンのことよ。えいえい、ってしてた時？」=“Thinking back to which part? Like when you were going at it?”
「顔ばっかり浮かんでくるかな？　可愛かったから。あとは、肌の温もりとか……」=“It's always just your face, I guess? You were seriously adorable. Also, your warmth and whatnot...”
「だからぁ、意識させないでったら」=“I told you, don't make me think of it, dummy.”
「……ごめん」=“...Sorry.”

> 「そ、そんなに真面目に謝らなくてもいいわよ。意識してくれるの……嬉しいし」=“Y-You don't need to apologize so seriously. It does me... happy, that you think about it.”

Reviewer: Previous line here was a lot better.
```suggestion
「そ、そんなに真面目に謝らなくてもいいわよ。意識してくれるの……嬉しいし」=“You don't have to apologize so earnestly. I'm happy... you're thinking about me.”
```
